packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
-import { observer } from 'mobx-react';
 import { Text, Icon, PageOverlay, DesktopWrapper, MobileWrapper } from '@deriv/components';
 import { routes } from '@deriv/shared';
 import { Localize, localize } from '@deriv/translations';
 import { useHistory } from 'react-router-dom';
+import CFDInstrumentsLabelHighlighted from './cfd-instruments-label-highlighted';
```

Comment:
Put your imports from libraries on top, then `@deriv` imports and the internal imports after them